src/components/RestaurantDetailModal/RestaurantDetailModal.jsx

```diff
@@ -1,18 +1,23 @@
 import "./RestaurantDetailModal.css";
 
-function RestaurantDetailModal() {
+function RestaurantDetailModal({ restaurant, onClose }) {
```

Comment:
```suggestion
function RestaurantDetailModal({ restaurant: { name, description }, onClose }) {
```
이런 식으로 구조 분해 할당 문법 사용하고 접근할 때 `restaurant.name` ➔ `name` 바로 접근할 수도 있을 것 같아요!🙃
(요건 개인 취향 차이인 것 같슴다)

Reply:
오 앞서 피드백해주신 RestaurantList와 조금은 다른 형태로 구조분해할당이 들어가네요! 좋은 방법인 것 같아용 😆